src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/layout.html.twig

```diff
@@ -53,8 +53,8 @@
                             </li>
                         </ul>
                     {% endif %}
-                    {{ render(path('_profiler_search_bar')) }}
-                    {% include '@WebProfiler/Profiler/admin.html.twig' with { 'token': token } only %}
+                    {{ render(path('_profiler_search_bar', app.request.query.all)) }}
```

Comment:
We never use `app` in the profiler as it's not always available (in Silex for instance). `request` should be available (at least that's the case in 2.8 where we are going to merge this PR).. which means that you should probably close this PR and open a new one on the 2.8 branch.